test/unit/tasks/cms/cms_test.rb

```diff
@@ -22,4 +22,31 @@ def test_fix_pagination_href
       assert_equal false, @page.published.include?('{{ part.link }}')
     end
   end
+
+  class FixInvalidSysNameSections < ActiveSupport::TestCase
+    def setup
+      @provider = FactoryBot.create(:provider_account)
+    end
+
+    test 'fixes invalid system names' do
+      section = FactoryBot.build(:cms_section, partial_path: '/', parent: @provider.sections.root, title: 'New section',
+                                 system_name: 'invalid name')
+      section.save!(validate: false)
```

Comment:
If this doesn't have a chance to fail, we could use `save` instead:

```suggestion
      section.save(validate: false)
```

Reply:
Commited